### ISSUE TITLE
This is retarded

### ISSUE THREAD

**Issue by woohooatb** [CoralineAda/postmeritocracy#22]
>But meritocracy has consistently shown itself to mainly benefit those with privilege, to the exclusion of underrepresented people in technology. The idea of merit is in fact never clearly defined; rather, it seems to be a form of recognition, an acknowledgement that “this person is valuable insofar as they are like me.” 
Source: https://postmeritocracy.org/

If you have to become a doctor, you have to study hard for it. Not anyone can be a doctor no matter the color of their skin or their gender. Pull this shit in medical or aviation field and see what happens. 

Thanks for ruining Linux though. You people will go to hell if such thing exist. Good luck with your regime while it last.

**Comment by michael-brade** [comment]
Fully agree. The way it is written now is a disaster. However, I like the (possibly initial) idea of it: be compassionate, don't be an asshole. But then don't throw the baby out with the bath water, i.e., keep the meritocracy. *Of course* it is not possible to define merit globally and exactly. It depends on the situation and the task at hand. If you have to do surgery or assemble a computer, it is rather easy to define it objectively. If you have to write a good program, it starts to get harder to define "good". And if you have to teach and care for people, it is even more difficult - maybe it is not even possible in this case. Meritocracy seems to be the solution: if many people agree that your results are good and they like you and your work, *something* was done right.

So to put it much more nicely, here's how I would like to write it:

 * keep the people who have proven they are capable, but teach them what it means to be civilized, honest, compassionate humans (if they are not)
 * let others join open-mindedly, let them show you what they are capable of and what they love to do
 * get rid of those that are not able to do the task at hand properly or those that are rude or mean or dishonest etc.
 * don't judge by opinion, religion, belief, color, looks, etc.
 * don't force people to push their limits, let them have a life as well
 * it is ok to say "no" to the wrong, to stop the lies

It may not be the best example, but it is one that many people know: look at those casting shows, like AGT or BGT or talent in general. They are getting it pretty right. It is not enough to be nice to win, or to be white or black or beautiful or ugly - it doesn't matter much at all - you actually have to have talent and be entertaining to be successful (in most cases). But it also shows that talent alone is not enough. If the person is a jerk, arrogant, etc., they get the boot.

You know, in the end I wonder why we even need to mention that we should be loving, compassionate humans. It should be natural. For some reason, it isn't.

**Comment by aschrijver** [comment]
> You know, in the end I wonder why we even need to mention that we should be loving, compassionate humans. It should be natural. For some reason, it isn't.

Yes.. see some of the issues in this repo :slightly_frowning_face: 

I like your points @michael-brade and think adding some of this would indeed be an improvement to the manifest. But the intentions are good. I don't know why they elicit such strong reactions from others. Maybe they are reading things I overlooked.

**Comment by michael-brade** [comment]
Ok, I have read it again, this time with a bit more distance. So now, here's what I think should be changed or dropped:

> We can add the most value as professionals by drawing on the diversity of our identities, backgrounds, experiences, and perspectives. Homogeneity is an antipattern.

*most value* -> drop "most". It doesn't make sense. If you write a piece of software on which people's lives depend, it is, frankly, completely irrelevant how diverse you are. Just don't make a mistake--or test properly and get rid of the mistakes before it goes live. And if that requires some people who believe their only value is technical, take them. We do add value by bringing a positive spirit into the environment, which then enables others to do their best. But that is certainly not the *most* important thing. Like I said, don't throw the baby out with the bath water. 

And what does "Homogeneity is an antipattern" even mean? If I have a team with project leaders, developers, testers, advertisers etc., and my developers are all on the same page regarding what they want, how they solve the problems and which tools they use, is that a problem? If so, why? I already have other non-technical people anyway. Then what is homogeneity?

What I said in the beginning is kind of complemented by another bullet point:

> We have an ethical responsibility to refuse to work on software that will negatively impact the well-being of other people.

...which can and should then be turned around as well: We have an ethical responsibility to make sure that the software we do work on will not negatively impact the well-being of other people. (e.g., by being incapable, neglecting, or not caring)


> We must make room for people who are not like us to enter our field and succeed there. This means not only inviting them in, but making sure that they are supported and empowered.

Yes, *iff* those people share the passion for our field *and* are capable of contributing. I had a person come to a job interview as a software developer, and that person said right away, "I don't like writing software, I don't program at home in my spare time - are you crazy?! - I want to read books and sit in the sun. I just do it because it is a job and I need the money." Fair enough, but now don't judge me if I say no to that. I don't *have* to make room for people like that, but I might if I see something in them, and/or if they are actually really good at programming nonetheless.

**Comment by michael-brade** [comment]
Oh! I just realized: all of those points in "postmeritocracy" -- how do you intend to test them? Certainly not by any objective, automated examination, right? So that leaves.... yes, meritocracy! Or does it? :smile: Meritocracy, extended to more than just technical abilities.

**Comment by aschrijver** [comment]
Good points, once again! I agree dropping '_most_' in 'most value' would be an improvement.

>And if that requires some people who believe their only value is technical, take them.

This is no problem in current text. It only states that non-technical and interpersonal skills are valued equally. But for every person these skills are on a balance. Doesn't necessarily mean the IoT programmer needs the same interpersonal skills as the CEO. But if he/she is a 'toxic person' that will probably have to outweigh his technical prowess.

> And what does "Homogeneity is an antipattern" even mean?

I was unclear on that too, and think it should be either clarified or omitted. I read homogeneity in the more holistic meaning of the word. Humans as a whole are better off if they operate in groups that are diverse in nature.

I worked in a IT scale-up with people from over 60 countries. It was really refreshing to hear all their different viewpoints and perspectives (on all aspects of life) and it did a lot of good to our ability to innovate. If, say, you'd build a team from an echo chamber that you yourself are part of, and e.g. do a project in machine learning, then you shouldn't be surprised if the AI you built became overly biased.

> We have an ethical responsibility to make sure that the software we do work on will not negatively impact the well-being of other people.

Agree. Do you think both should be mentioned, or only the - more positive - reversed rephrasal?

> Yes, iff those people share the passion for our field and are capable of contributing.

Sure, but that is beyond the scope of this text. You should be accommodating to newcomers, give them a chance. If they do not take it, that is their choice. Your responsibility ends there, you should not be obliged to keep trying to get them on the right path (unless you so choose).

> Oh! I just realized: all of those points in "postmeritocracy" -- how do you intend to test them? 

You can't. But this is true for any ethical and moral framework. You appeal to common sense and describe a direction towards a common good. But this is subjective.

I have to admit the name 'Post-meritocracy' to what was meant with that term. And I do not think the term should be used. This manifesto better be renamed. We want to go beyond the current meaning or implementation in practice of meritocracy, but - as you rightfully point out - this does not mean meritocrarcy disappears. We should just strive to a better interpretation of it. A new meritocracy.

Brainstorming for a better name, that might stick (?):

- The Genuine Meritocracy Manifesto (aka the GEM manifesto :)

**Comment by michael-brade** [comment]
Yes, lovely, I also agree with everything you wrote :)

> This is no problem in current text. It only states that non-technical and interpersonal skills are valued equally. But for every person these skills are on a balance. Doesn't necessarily mean the IoT programmer needs the same interpersonal skills as the CEO. But if he/she is a 'toxic person' that will probably have to outweigh his technical prowess.

*Exactly*! I think if we go up the positive side of qualities, then it depends on the job if interpersonal or technical skills are the most important. But if we go down the negative, toxic side, then suddenly the interpersonal skill becomes the dominant one.


> I worked in a IT scale-up with people from over 60 countries. It was really refreshing to hear all their different viewpoints and perspectives (on all aspects of life) and it did a lot of good to our ability to innovate. If, say, you'd build a team from an echo chamber that you yourself are part of, and e.g. do a project in machine learning, then you shouldn't be surprised if the AI you built became overly biased.

:joy: Indeed.


> > We have an ethical responsibility to make sure that the software we do work on will not negatively impact the well-being of other people.
> 
> Agree. Do you think both should be mentioned, or only the - more positive - reversed rephrasal?

I think the more positive is enough, but I wouldn't mind keeping them both for emphasis. Then we'd explicitly cover software systems that are used for killing people - and I refuse to work on those.

> > Yes, iff those people share the passion for our field and are capable of contributing.
> 
> Sure, but that is beyond the scope of this text. You should be accommodating to newcomers, give them a chance. If they do not take it, that is their choice. Your responsibility ends there, you should not be obliged to keep trying to get them on the right path (unless you so choose).

Yeah, absolutely. Now let me ask: in current meritocracy, isn't that *already* the case? To be honest, I didn't really understand the criticism on Wikipedia, and I don't agree with the parts I do understand. Just one example: if it turns out that to some OSS project mostly male people contribute, then I wouldn't blame meritocracy for it. I would look at how people interact and if they allow new potential contributors to join. It may just be that that project is an area women are not interested in. Then it would be totally ok, I wouldn't see a need to change anything - although they might attract women anyway if they would radiate joy and being a cool, sexy group :sunglasses:  Or it may be that the people there don't like women and/or don't want to work with them. Or that they are too emotional and judgemental in a way that turns women off. Then I still don't think meritocracy is the problem, it's the people. So it wouldn't help to force them to work with women, that wouldn't solve the root problem. You might just get rid of these people, but that, too, wouldn't really help IMHO. The best would be to actually develop consciousness, compassion, and to look at one's own resistances and fears, and see if they are really needed. It may be the more difficult way, but it's certainly the much more rewarding one. I think Linus may become a good example for that.
 
> I have to admit the name 'Post-meritocracy' made me curious to what was meant with that term. And I a better term should be used. This manifesto renamed. We want to go beyond the current meaning and implementation in practice of meritocracy, but - as you rightfully point out - this does not mean meritocracy disappears. The manifesto offers a better interpretation of it. A new meritocracy.
> 
> Brainstorming for a better name, that might stick (?):

Good idea! But I'm not really good at finding names (yet...).

> * The Genuine Meritocracy Manifesto (aka the GEM manifesto :)

well, it's a good start. Doesn't make me go "wow" yet :innocent:

**Comment by xCuri0** [comment]
I f*cking hate anyone who discriminates straight males
Thats why I don't like this code of conduct because it was written by some trap who does

**Comment by aschrijver** [comment]
It is good that you pointed to Wikipedia. I must confess was a newbie on the concept, and I am learning now as I go. Had a different idea about it. What I found interesting in the WP page are:

> _"The term "meritocracy" was originally intended as a negative concept. [..] The ‘meritocracy’; the society in which the gifted, the smart, the energetic, the ambitious and the ruthless are carefully sifted out and helped towards their destined positions of dominance."_ (paraphrased from [Wikipedia](https://en.wikipedia.org/wiki/Meritocracy))

With regards to your example: _"an area women are not interested in"_ I fully agree meritocracy is not in play. But _"may be that the people there don't like women and/or don't want to work with them"_ is more interesting. There could be valid reasons to have such a group, as long as meritocratic motives are not in play, I guess it is alright. But the lines can get blurry quickly.

What you see often these days, is that women - who want to (rightfully) break the glass ceiling, and get to be treated equal wrt their male peers - are organizing female-only initiatives, meetups, conferences, etc. The goal is to take away barriers and empower more women to reach above and beyond to - what they perceive as - their own limits. I can understand this activism, and it comes forth from an inequality that does exist. But at the same time it would seem to violate this manifesto, doesn't it?

> So it wouldn't help to force them to work with women, that wouldn't solve the root problem. You might just get rid of these people, but that, too, wouldn't really help IMHO. The best would be to actually develop consciousness, compassion, and to look at one's own resistances and fears, and see if they are really needed. [..] And maybe that is the intention of this new manifesto; I would like that.

Yes, agreement once again :smiley:  . This manifesto - if read and understood by people in that group - would contribute to that.

> >a better term should be used. This manifesto renamed.
>
> Good idea!

I've come to see that maybe 'Post-Meritocracy' isn't so bad a choice of words after all.

> _"Meritocracy is a political philosophy which holds that certain things, such as economic goods or power, should be vested in individuals on the basis of talent, effort, and achievement, rather than factors such as sexuality, race, gender, or wealth."_ (source [Wikipedia](https://en.wikipedia.org/wiki/Meritocracy))

With this definition you can see how this concept is really embraced by companies and corporations in our modern society. Hire best people, become most competitive, gain your place among _the winners_ in this world, the definition of success, etc. We are having our value systems all wrong.

I think there is an increasing awareness of this fact, and meritocracy plays a part in this. I am personally convinced that our economic systems are failing us - even though this is still fiercely denied by many people (with current _winners_ at the front). This, I think, is a root cause of many of humanity's big problems (environmental destruction, consuming at the level of 2 earths, etc.).

The new meritocracy that we should strive for, should lead to a rebalancing of values, of a redefinition of success. One leading to sustainability in a holistic (not just environmental) sense. Some of this is already reflected in current text. Some of it is not.

Am I taking this too far (especially given that this manifesto is scoped to the IT world)?

PS @CoralineAda would it be possible for you to rename this issue to something more constructive?

**Comment by michael-brade** [comment]
> > _"The term "meritocracy" was originally intended as a negative concept. [..] The ‘meritocracy’; the society in which the gifted, the smart, the energetic, the ambitious and the ruthless are carefully sifted out and helped towards their destined positions of dominance."_ (paraphrased from [Wikipedia](https://en.wikipedia.org/wiki/Meritocracy))

Well... you know, things change. I don't really care if a word was intended to mean one thing some 60 years ago if right now we use it for something else. I wouldn't use a decades old intention for not doing something today.

Furthermore, I also wouldn't care much about the word itself, it's just a reference for something. If we want a different word, so be it. Much more interesting is that which we are referring to. And when I look at OpenSource, then I firmly believe that what we are doing at the moment kind of works. There is always room for improvement since nobody is perfect, but I wouldn't throw the whole thing out the window.


> With regards to your example: _"an area women are not interested in"_ I fully agree meritocracy is not in play. But _"may be that the people there don't like women and/or don't want to work with them"_ is more interesting. There could be valid reasons to have such a group, as long as meritocratic motives are not in play, I guess it is alright. But the lines can get blurry quickly.
> 
> What you see often these days, is that women - who want to (rightfully) break the glass ceiling, and get to be treated equal wrt their male peers - are organizing female-only initiatives, meetups, conferences, hackathons, etc. The goal is to take away barriers and empower other women to reach above and beyond to - what they perceived as - their own limits. I can understand this activism, and it comes forth from an inequality that _does_ exist. But at the same time it would seem to violate this manifesto, doesn't it?

Indeed, I agree. Except we are overdoing it.

* Where is the difference of doing female-only things and male-only things? If you look at it dispassionately, there is none. So both should be allowed, without one party feeling threatened by the other in either case.

* Men and women *are* different (the most obvious being the physical difference), so I would hate it if we tried to force them both into the same little box. Appreciate the differences! How boring would the world be if there were no differences?

* The problem we have these days is not really the differences. It's people judging the differences, and then taking away opportunities according to the judgements. This is what we call discrimination. So all we need to do is look at where that happens and simply stop it.

Instead, we overdo it, we go to the opposite extreme: we set up quotes (arbitrary numbers with no relation to reality, see below), we try to have the same amount of women as there are men in all "desirable" positions. And if we only find a man and cannot find a woman that fits, we leave the position open or use a women that doesn't fit just to match the quote. What is that?! That is just the same discrimination, but now against men. How is that any better? Equal opportunities should not mean everybody has to *be* equal. It's simply not possible. And just for a moment, if we do think that is the right way, then why use only the gender for reference? Why not also a quote for the intelligent vs dumb people? We could set up an IQ test and require that in any given position there is an equal amount of people with an IQ < 100 and IQ > 120. Right? Or the rich vs poor. Or people with blond hair vs brown hair. Well, you can see how flawed this is. And there are more examples: why do we need 50-50 men-women leading a company, but not 50-50 men-women sitting in prison? Or doing construction work? Both have a lot more men. And then, why not have a 50-50 quote for teachers in kindergarden level, or for midwives? You see, we should not judge the amount of people in a position, we shouldn't be biased, but we should instead give everyone a change to do what they would *love* to do.

And here is why I said arbitrary numbers: did we ever count the number of women and men? Do we really have the same amount of women as we have men? And if so, did we count the number of men and women who studied maths, computer science, medicine, art, business administration, etc.? Is that really all 50-50? Hardly, so where does that 50-50 come from? It's unrealistic, isn't it? So if we want to do it properly, and if we do want quotes, we should count, only then would we establish equal probabilities and opportunities.

> I've come to see that maybe 'Post-Meritocracy' isn't so bad a choice of words after all.
> 
> > _"Meritocracy is a political philosophy which holds that certain things, such as economic goods or power, should be vested in individuals on the basis of talent, effort, and achievement, rather than factors such as sexuality, race, gender, or wealth."_ (source [Wikipedia](https://en.wikipedia.org/wiki/Meritocracy))
> 
> With this definition you can see how this concept is really embraced by companies and corporations in our modern society. Hire best people, become most competitive, gain your place among _the winners_ in this world, the definition of success, etc. We are having our value systems all wrong.

Hmm. Now I don't know anymore if I agree or not :smile: If I want to find a president, I wouldn't want to choose based on sexuality, race, gender, or wealth. I would want to choose based on talent, honesty, and how much the person cares for the people. Isn't that Meritocracy? These days it's different, it's not so much about talent and honesty, it's all about money.

**BUT** I agree, we have our value systems all wrong! Right now we are all about money, and we define desirable positions as those that give money and power. And everyone wants money, so everyone wants those positions, so we fight for them. And those of us that already have enough still want more. This is so wrong. We (soon) could have the machines do most of the work needed to keep us alive, give everyone a roof over their head and enough food, and then add whatever more is available to those who insist on fighting for it or who actually contribute significantly to society. Instead, we allow a few people to have *a lot* more than they need for doing almost nothing while we let others work like slaves and starve. And all in the name of money.

So: 

> The new meritocracy that we should strive for, should lead to a rebalancing of values, of a redefinition of success. One leading to sustainability in a holistic (not just environmental) sense. Some of this is already reflected in current text. Some of it is not.

I **absolutely agree**!

> Am I taking this too far (especially given that this manifesto is scoped to the IT world)?

No :smile: not in my opinion. Only in so far as we probably won't solve those problems here :stuck_out_tongue_closed_eyes:

**Comment by aschrijver** [comment]
> Much more interesting is that which we are referring to. And when I look at OpenSource, then I firmly believe that what we are doing at the moment kind of works.

There were some interesting discussions on Hacker News recently about OSS missing the mark, because of the big corporations (e.g. Amazon) incorporating the cool projects without giving back sufficiently and properly honoring the creators of these projects who put their sweat and tears in it. (Sorry, no links, I have to look that up still and read more about it. But it was interesting. Also some discussion of 'evil' licensing additions that are breaking OSS ideas).

> Indeed, I agree. Except we are overdoing it.

The points you mention were also what I had in mind with regards to your bullet points, when I wrote _"at the same time it would seem to violate this manifesto, doesn't it?"_. But indeed, in the quest to improve things, we (the general form) tend to overdo it. Your examples are correct.

I think it is important that any 'post-meritocracy' text cannot be interpreted that way. Should be carefully construed to not give that impression. (Probably now it does have controversial text, given the reactions to it.. also did a lookup on previous HN discussion, where you'll find more of that).

> Hmm. Now I don't know anymore if I agree or not

I am all for hiring the best people, with matching skills, but measued against the right set of values. So I am not suggesting positive discrimination, or anything like that. Your **BUT** (single T :stuck_out_tongue_closed_eyes: ) is exactly what I am talking about wrt values. You can see how with the current definition of meritocracy the 'rise of the machines' will be detrimental (probably, but opinions vary) to the human workforce.

> Only in so far as we probably won't solve those problems here stuck_out_tongue_closed_eyes

Anyway I enjoyed our little discussion, @michael-brade :smiley:  . Don't know, maybe @CoralineAda wants to jump in, and we can brainstorm some changes. But maybe you'd also like to join the [Humane Tech](https://humanetech.com] community [forum](https://community.humanetech.com). We are always looking for critical thinkers that do not shy away from a good discussion. No problem if you are not interested, though, of course.

**Comment by michael-brade** [comment]
Yes, I enjoyed it very much as well! :smile: We'll see where that leads to.

Now, I am actually very impressed by Humane Tech! Good ideas and great to see people working on it. I also had a look at the forum, and I wouldn't say I shy away from discussion if I can contribute... but I feel I am at my limit at the moment. To follow the posts on the forum *and* to contribute properly, I would have to spend much more time than I have right now. So for now I will remember it and see if my situation changes by December/January.

**Comment by aschrijver** [comment]
No problem at all. Just happy you are aware of the community :)

**Comment by michael-brade** [comment]
Oh my God! I finally read the new Linux Code of Conduct and now I understand why people are angry. It went from one extreme to the other. Suddenly you can get rid of projects, people, and companies you don't like--not because you can actually do better but simply because you now have political leverage. E.g. now you can submit a flawed patch and say: "they weren't nice, I'm a minority and they rejected my contribution!" Or you can annoy others long enough until they say a wrong word and then get them on that.

I was just referred to this **great** video with a really succinct explanation of what's wrong with CoCs: https://www.youtube.com/watch?v=nND3EYzIONg

And it seems this is not just phantasy or unfounded fear. See here what Coraline herself did, for instance:

https://github.com/opal/opal/issues/941
https://twitter.com/CoralineAda/status/1029170073938944000 (double standards?)
https://twitter.com/CoralineAda/status/1041441155874009093

**Comment by aschrijver** [comment]
Great, that you bring that to attention @michael-brade ! I'll not take any opinion or side on all of this, but it saddens me to see all the anger and frustration, and - probably - misunderstandings. The issues underlying all of this are complex.

I am, however, convinced that technology, and the way it alters human communication, plays a large role in the severity and tone of these discussions. That is very much a Humane Tech subject of interest, and I will bring it to the attention of our community.

An interesting statement by Mozilla regarding Meritocracy, led me to reach out to Larissa Shapiro - Head of Global Diversity and Inclusion at Mozilla.

This is the blog post I was referring to: https://blog.mozilla.org/inclusion/2018/10/02/words-matter-moving-beyond-meritocracy/

The whole topic of Meritocracy and controversies surrounding it, obviously warrant further investigation and approaches to bring solutions.

**Comment by michael-brade** [comment]
Thank you for that link. It is sad to see this happen, again, without any proof for this opinion, for she judges open source and meritocracy as discriminative, harassing, and identity-based. 

Now give me some actual examples of prominent open source projects where this is the case. And then give me some research that actually did the numbers and came to the conclusion that in open source these problems are more prevalent than in other software projects (as she claims). I have not yet seen any of these statements proven, but maybe you do have a link...?

On the other hand, I do have seen these social justice things punish and exclude people, sometimes even destroy whole projects. So if in projects with CoCs more people are excluded than in projects without CoCs, wouldn't it be fair to say that the net-effect of "being inclusive" is exclusive and divisive? 

So I agree with you when you say "..obviously warrant further investigation". I think it is about time someone does that research and collects the statistics--just like I mentioned earlier already. We need real, true numbers to be able to make the correct decisions.

**Comment by aschrijver** [comment]
> So I agree with you when you say "..obviously warrant further investigation". I think it is about time someone does that research and collects the statistics

FYI I've just brought this to the attention of our Humane Tech Community: https://community.humanetech.com/t/meritocracy-and-the-complex-diversity-inclusion-issues-surrounding-it/2828

**Comment by tonylambiris** [comment]
I'm still having a hard-time wrapping my head around this whole movement (to clarify, the the sum of it's entirety, not specifically this one or any other).

I suppose the main issue I'm having is where to even start. Most of the stuff I've read have almost been exclusively covering the broad scope (usually a formal interview with some backstory/history on the person being interviewed), or covering a super narrow blip of a specific event that has typically required a sizable investment of time backtracking through thread posts, mailing lists, or jumping back in forth between searches of the subjects involved to paint a more complete portrait of the timeline of events that occurred immediately before and/or after. The third and loudest in terms of frequency and vitriol is on Twitter, and although I don't hold lend individual tweets or conversations with much gravitas it does pepper in some of the finer details and nuance due to how much looser people tend to be on there (ie: these tweets don't express the thoughts of my employer, etc).

Is the main purpose to have a general manifesto of Best Behaviors or is this more of a think-tank project that eventually would like to institute something akin to a board of advisers or perhaps a more loosely-knit fellowship comprised mostly of tech-titans, meaning people separately and privately employed by the major players in the industry (ie: higher-ups employed by Google, Microsoft, Amazon, Facebook, etc...). Has anyone thought that far ahead or better yet is there any information that could comprise for a rough timeline with specific milestones?

TL;DR all the information available is either incredibly vague or super specific, so where can one start on the specifics to turn the squelch knob up on the noise?

Thanks in advance.

**Comment by Hex-GHI** [comment]
If one wants to become a doctor but doesn't study or care about work ethics or quality, one's patients will suffer due to one's own ineptitude. The pertinent question is - will the doctor's lay patients realize their diminished health status is due to their doctor's fault as opposed to their own lifestyle choices? The same is true for lawyers. Both of these professions require skills and epistemic knowledge that must be learned in a comprehensive curriculum that includes human interactions and professors; access to symbolic knowledge is simply not enough. Google may be the open window to all our collective knowledge, but few lay people ever become self-taught doctors or lawyers by mere exposure to Google.

The further we move down from the top of that pyramid of professionalism, the more likely said lay person is to be able to identify where the fault actually lies. If a waiter serves food that's been accidentally dropped on the floor and put back on the plate, the consumer will know that there's something wrong with the food (there's dirt and hair it now). The consumer knows his lack of hunger isn't the problem. That's because most everyone is "well rounded", at least to some degree, but the complexities grow as the profession specializes. The more we specialize into any field of knowledge, the less we have time to learn the details of anything else. Einstein had a full closet of the same suit so that he wouldn't have to make that daily decision on what to wear. On the other hand, the more one tries to be well-rounded, the less details and skills one may learn about anything in specific. A jack of all trades is the master of none. We, as humans, can be no more than the product of our biology and life's experiences. None of us have ever lived any part of our lives in another's shoes.

The idea of merit is not clearly defined in the field of technology, because technology is still a largely unexplored frontier. Software designers are considered valuable when they duplicate (or exceed) the achievements of another who first created that same innovation. This is a matter of human nature, not a matter of the technology itself. We admire those who can achieve what we can achieve because we can relate to them; this creates epistemic bubbles and echo chambers. Arnold Schrijver (above) posted an on point CHT article. https://philpapers.org/rec/NGUECA "An epistemic bubble is a social epistemic structure in which other relevant voices have been left out, perhaps accidentally. An echo chamber is a social epistemic structure from which other relevant voices have been actively excluded and discredited. Members of epistemic bubbles lack exposure to relevant information and arguments. Members of echo chambers, on the other hand, have been brought to systematically distrust all outside sources."

I note that Tony Lambiris asked "is this... a think-tank project that eventually would like to institute something akin to a board of advisers or perhaps a more loosely-knit fellowship" Indeed, I believe the answer to this problem is the regulation of a non-profit ecosphere in cyberspace (to prevent concentrations and abuses of power that come from that field and from the bad side of our human natures). To answer "where can one start on the specifics," I have drawn on all my life's experiences and skills to provide said model for such an open source decentralized union which values everyone's merit equally and lawfully. http://airofusion.com/ghi/ I remain resolute with my CHT campaign to turn this laudable dream into a democratic reality for all of us. But because my background is self-taught law (and dealing with continuing systematic attacks by powerful lawyers and judges, not software development), and because I have no network of like minded people or any meaningful social skills (or even any experience with Facebook, Twitter, and social media in general), I am aware that I am taking a huge risk of not being accepted (as I have no peers).

With that in mind I note that the manifesto says "We must make room for people who are not like us to enter our field and succeed there. This means not only inviting them in, but making sure that they are supported and empowered." Yet as Michael-Brade (above) conversely noted, this should only be true "if those people share the passion for our field and are capable of contributing." The software developer who doesn't like writing software, and doesn't program at home in his spare time in order to read books and sit in the sun is essentially saying that the employer should hire him with pay, with or without any software development contributions, because he's got a technology network of professional and social skills. He's essentially saying "I'm a bad ass software developer and I know it. Me and my network of friends are like rare magicians, so if you don't hire us you'll never figure out the illusion and therefore you'll lose profit." These are what we call toxic people with bad human natures. How do employers and the open source community differentiate between the intentions of people like me, versus the bad ass software developer, if those employers have never had any human interaction with said applicants and no set of formal laws to regulate the two of us in this same field of technology exists? Without those formal laws to regulate us evenly, the only things that can be considered are their underlying networks of friends and skills. Indeed, even the United Nations now has an Internet Governance Forum whose objectives are to resolve these issues on a global scale.
